### PR TITLE
Add Chennai to Master

### LINKED ISSUE #19: Passion4aix - Chennai, India
## Chennai, India

- [x] Create a branch
- [x] Add file
- [x] Create a commit
- [ ] Open a PR
- [ ] Have a discussion about PR
- [ ] Merge PR

### PR DESCRIPTION
**Pull request by passion4aix**
Closes #19

### CODE REVIEW

**Comment by passion4aix** [comment]
@githubteacher

File: places/passion4aix-chennai.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+## Chennai, India
+
```
Comment (aldorian2015) [inline places/passion4aix-chennai.md]: I would love to go to India!

Comment (passion4aix) [inline places/passion4aix-chennai.md]: most welcome :)

Comment (passion4aix) [inline places/passion4aix-chennai.md]: :+1:

**Pull request merged**